### PR TITLE
MINOR: Add explicit result type in public defs/vals

### PR DESCRIPTION
**Pull request by highluck**
specifying the type will reduce mistakes.

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: core/src/main/scala/kafka/network/RequestChannel.scala [hunk 1]
```diff
@@ -76,11 +76,11 @@ object RequestChannel extends Logging {
...
-    @volatile var requestDequeueTimeNanos = -1L
-    @volatile var apiLocalCompleteTimeNanos = -1L
-    @volatile var responseCompleteTimeNanos = -1L
-    @volatile var responseDequeueTimeNanos = -1L
-    @volatile var apiRemoteCompleteTimeNanos = -1L
+    @volatile var requestDequeueTimeNanos: Long = -1L
+    @volatile var apiLocalCompleteTimeNanos: Long = -1L
+    @volatile var responseCompleteTimeNanos: Long = -1L
+    @volatile var responseDequeueTimeNanos: Long = -1L
+    @volatile var apiRemoteCompleteTimeNanos: Long = -1L
```
Comment (ijuma) [inline core/src/main/scala/kafka/network/RequestChannel.scala]: I don't think the type adds much for cases where a literal is used.

**Review by ijuma — COMMENTED** [review]
Thanks for the PR, one comment.

**Comment by ijuma** [comment]
ok to test

**Comment by ijuma** [comment]
ok to test

**Comment by ijuma** [comment]
ok to test

**Review by ijuma — APPROVED** [review]
LGTM

File: core/src/main/scala/kafka/metrics/KafkaMetricsConfig.scala [hunk 2]
```diff
@@ -29,12 +29,12 @@ class KafkaMetricsConfig(props: VerifiableProperties) {
    * Comma-separated list of reporter types. These classes should be on the
    * classpath and will be instantiated at run-time.
    */
-  val reporters = CoreUtils.parseCsvList(props.getString(KafkaConfig.KafkaMetricsReporterClassesProp,
+  val reporters: Seq[String] = CoreUtils.parseCsvList(props.getString(KafkaConfig.KafkaMetricsReporterClassesProp,
```
Comment (ijuma) [inline core/src/main/scala/kafka/metrics/KafkaMetricsConfig.scala:33]: The Scala 2.13 build failed due to this change. To make it work, you need to add `import scala.collection.Seq`.

**Comment by highluck** [comment]
@ijuma 
Thank you 
i'm code update!

**Comment by ijuma** [comment]
ok to test

**Comment by highluck** [comment]
@ijuma 
retest this please

**Comment by ijuma** [comment]
ok to test

**Comment by highluck** [comment]
@ijuma 
Thank you!
But Build is not triggered ㅜㅜ

**Comment by highluck** [comment]
@ijuma 
Thank you!
But Build is not triggered ㅜㅜ

**Comment by ijuma** [comment]
retest this please

**Comment by ijuma** [comment]
ok to test

**Comment by ijuma** [comment]
A few flaky test failures that are clearly unrelated since this PR did not change any behavior (just added type annotations).

**Review by ijuma — APPROVED** [review]
LGTM

**Comment by highluck** [comment]
@ijuma Thank you

**Pull request merged**